### ISSUE TITLE
AttributeError: 'EccPoint' object has no attribute '_point_get'

### ISSUE THREAD

**Issue by jiep** [Legrandin/pycryptodome#312]
* Step to reproduce

```
root@35ca8302c755:/# python -V
Python 3.7.4
root@35ca8302c755:/# pip install pycryptodome
Collecting pycryptodome
  Downloading https://files.pythonhosted.org/packages/4b/43/b6df7cd1354cc03c80cb972d5930b1adc209c717e9141883ca9026314eda/pycryptodome-3.8.2-cp37-cp37m-manylinux1_x86_64.whl (9.6MB)
     |████████████████████████████████| 9.6MB 2.3MB/s
Installing collected packages: pycryptodome
Successfully installed pycryptodome-3.8.2
root@35ca8302c755:/# python
Python 3.7.4 (default, Jul 13 2019, 14:04:11)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from Crypto.PublicKey import ECC
>>> key = ECC.generate(curve='P-256')
>>> T = -key.pointQ
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/Crypto/PublicKey/ECC.py", line 304, in __neg__
    result = _ec_lib.ec_ws_neg(np._point_get())
AttributeError: 'EccPoint' object has no attribute '_point_get'
```

In version `3.7.3` it seems to work properly:

```
root@35ca8302c755:/# python -V
Python 3.7.4
root@35ca8302c755:/# pip install pycryptodome==3.7.3
Collecting pycryptodome==3.7.3
  Using cached https://files.pythonhosted.org/packages/9e/55/038b18574705a86548e0008e4ea88a80393a370068e8a67c28becf2f4865/pycryptodome-3.7.3-cp37-cp37m-manylinux1_x86_64.whl
Installing collected packages: pycryptodome
Successfully installed pycryptodome-3.7.3
root@35ca8302c755:/# python
Python 3.7.4 (default, Jul 13 2019, 14:04:11)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from Crypto.PublicKey import ECC
>>> key = ECC.generate(curve='P-256')
>>> T = -key.pointQ
>>> T
<Crypto.PublicKey.ECC.EccPoint object at 0x7fcb2aba1fd0>
```

**Comment by Legrandin** [comment]
Thanks for reporting. Fixed on master with 9ad66cfdd12df14bc7c5e233da754aa87f147c61.

**Comment by jiep** [comment]
Thanks!
